build.py

```diff
@@ -0,0 +1,418 @@
+#!/usr/bin/env python
+# A commandline script using argparse that builds a vgcn image with packer
+# CAUTION: If a directory called 'images' exists in this directory, it will be deleted!
+# Required arguments are the template, which are named like the anaconda-ks.cfg files without the
+# anaconda-ks.cfg suffix and the provisioning, separated by space and named like the ansible-playbooks
+# in the ansible directory without the .yml suffix.
+# Optionally you can specify the path to the Packer binary you want to use (--packer-path)
+# the path to the conda env you want to use (--conda-env)
+# the path to your ssh private key for copying the images to sn06 and
+# make them publicly available (--publish)
+# you can also source your OpenStack application credentials first and set the (`--openstack`) flag
+# to create a raw image in your openstack tenant
+# If you have trouble with the ansible provider on your setup, you can specify additional
+# --ansible-args="..." to e.g. solve the issues with scp on some distros
+
+
+import argparse
+import datetime
+import os
+import pathlib
+import shutil
+import signal
+import subprocess
+import sys
+import threading
+import time
+
+DIR_PATH = os.path.dirname(os.path.realpath(__file__))
+
+STATIC_DIR = "/data/dnb01/vgcn/"
```

Comment:
You do not need to, but if you intend to use it exclusively as a path, maybe make it a `pathlib.Path` object from the beginning? This can help later (see other related comments).

```suggestion
DIR_PATH = pathlib.Path(__file__).parent.absolute()

STATIC_DIR = pathlib.Path("/data/dnb01/vgcn/").absolute()
```